tests/baselines/reference/user/npmlog.log

```diff
@@ -0,0 +1,103 @@
+Exit Code: 1
+Standard output:
+node_modules/npmlog/log.js(5,11): error TS2539: Cannot assign to '"/home/nathansa/ts/tests/cases/user/npmlog/node_modules/npmlog/log"' because it is not a variable.
```

Comment:
This error right here's a problem. It's not going to be the same anywhere else and the test is always going to fail. @sandersn should we be printing something else here?